src/parseTools.js

```diff
@@ -61,6 +61,11 @@ function preprocess(filename) {
   const isHtml = (fileExt === 'html' || fileExt === 'htm') ? true : false;
   let inStyle = false;
   const lines = text.split('\n');
+  // text.split yeild an extra empty line at the end if text itself ends with a newline.
+  if (text[text.length-1] == '\n') {
+    lines.pop()
+  }
```

Comment:
Though, checking `text` and popping `lines` seems less natural to me than checking `lines` and popping it? That is,
```js
if (lines[lines.length - 1] == '\n') {
  lines.pop();
}
```

Reply:
Yes I admit looks odd.. I thought i had good reason to do it this way but I'm not sure now.. will go with your version,